src/content/docs/en/recipes/add-yaml-support.mdx

```diff
@@ -4,16 +4,30 @@ description: Learn how you can import YAML data by adding a Rollup plugin to you
 i18nReady: true
 type: recipe
 ---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 Astro builds on top of Vite, and supports both Vite and Rollup plugins. This recipe uses a Rollup plugin to add the ability to import a YAML (`.yml`) file in Astro.
 
 ## Recipe
 
 1. Install `@rollup/plugin-yaml`:
-
-    ```bash
-    npm install @rollup/plugin-yaml --save-dev
-    ```
+    <PackageManagerTabs>
```

Comment:
This change is not related to the PR description and is inappropriate in this PR. Please remove and make a separate PR. (Note: I had to spend extra time figuring out this change because there was no original indentation issue in docs.)

Reply:
Reverted